transformers/storage/medianizer/initializer/initializer.go

```diff
@@ -0,0 +1,16 @@
+package initializer
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+	mcdStorage "github.com/makerdao/vdb-mcd-transformers/transformers/storage"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/storage/medianizer"
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/storage"
+)
+
+var medianizerAddress = constants.GetContractAddress("MCD_MEDIANIZER")
```

Comment:
Is there a way to handle getting two different addresses here besides creating a config for `ETH_MEDIANIZER` and `BAT_MEDIANIZER`